CONTRIBUTING.md

```diff
@@ -47,7 +126,7 @@ This will clone the main repo to a local repo on your disk and set up the `origi
 Next you will set up the the second side of the triangle to your fork repo.
 
     cd osgi.enroute.site
-    git remote add fork git@github.com:github-user/osgi.enroute.site.git
+    git remote add fork https://github.com/<YourUserName>/osgi.enroute.site.git
```

Comment:
Well GitHub does seem to recommend https: https://help.github.com/articles/which-remote-url-should-i-use/

But if you have 2FA for your your github credentials (and you do, dont you?), then you will need create a personal access token to use instead if your actual github password. 

At this point, ssh is a lot better. :-)